### ISSUE TITLE
First alpha release

### ISSUE THREAD

**Issue by lukasgraf** [plone/plone.restapi#111]
As discussed at the Barcelona Sprint, there's going to be a first alpha release of `plone.restapi` soon.

These are the current blockers that are left:

- [x] **JWT based authentication** (PR #109)
- [x] **Batching** (PR #99)
- [x] **File / Image download** (issues #9 / #10, *needs discussion*)

---

**Deferred for now:**

- **"Framing" the default `GET` response** differently, with fields in their own nested dictionary (issue #6, *needs discussion*)

For reference, this was the state of the Kanban board was in when we left the sprint yesterday:
![kanban_barcelona](https://cloud.githubusercontent.com/assets/405124/15448835/3273cc00-1f6d-11e6-9759-c9dd3b352ecd.jpg)

@tisto @buchi @sneridagh did I forget anything?

**Comment by tisto** [comment]
@lukasgraf I think you listed everything. We can work on the actions right after the first alpha release...

**Comment by buchi** [comment]
@tisto @lukasgraf I would suggest to make a 1.0a1 release now.
The Framing and File/Image download changes are compatibility breaking and I would like to have 'em in a changelog. Thus I propose to do this in a later release.

**Comment by lukasgraf** [comment]
@buchi Framing has already been deferred.

Regarding breaking changes, we've already had plenty of those (`members` -> `items`, `items_count` -> `items_total` (Search), the introduction of batching, `search` -> `@search`, just to name a few).

But I'm fine with deferring File/Image download as well until after the release.

**Comment by tisto** [comment]
@buchi @lukasgraf I would actually like to wait for the file/image changes. I don't think implementing @lukasgraf's proposals is that much work.

**Comment by buchi** [comment]
@tisto @lukasgraf I'm fine with waiting for the file/image changes, but then I really would like to have a changelog to document changes.

**Comment by lukasgraf** [comment]
@buchi file/image changes are ready for a review, just didn't want to have it merged if I you felt strongly against having those changes in the first release already. Will open a PR right now.

**Comment by tisto** [comment]
All open issues for a first alpha release have been closed / merged. Expect a first release soon!

**Comment by lukasgraf** [comment]
@tisto excellent, thank you! 🎉  

I noticed that the RTD docs build is currently failing again - docs for some features that already have been merged into master (like JWT auth and batching) haven't been published yet.

I briefly tried triggering a build manually that failed because [RTD was missing `sphinxcontrib.httpdomain`](https://readthedocs.org/projects/plonerestapi/builds/4054889/). We removed that dependency during the sprint, because it wasn't / isn't being used, but it seems RTD tries to reuse build environments to some extent. Maybe we should just re-add it?

The other failure [(`TypeError: isinstance() arg 2 must be a class, type, or tuple of classes and types`)](https://readthedocs.org/projects/plonerestapi/builds/4049140/) is, IIRC, a version conflict that's flaky / not exactly deterministic - during the sprint I was able to work around that one by just manually triggering the build again (didn't have the nerve to look into it any deeper).
